pom.xml

```diff
@@ -587,7 +587,26 @@
       </plugins>
     </pluginManagement>
     <plugins>
-
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
+        <version>2.3.0</version>
+        <configuration>
+          <javaVersion>8</javaVersion>
+          <ignoreClassNamePatterns>
+            .*.Input.*,.*Test,.*BadCommas
```

Comment:
I understand `Input` exclusion. Can you clarify why there are the other 2 and if they are meant to be removed in a future PR?

Reply:
Unfortunately, there is no way to exclude some sources by folder, only by class name.
it is a leftover, will fix it

Reply:
If you will fix it, then that means this PR will not close the issue as your first says?

Reply:
nevermind, I removed this part